.github/workflows/ci.yml

```diff
@@ -68,6 +68,10 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: All tests
+        env:
+          PDE_ORG_RESULTS_AGGREGATOR_UPLOAD_USER: ${{ vars.PDE_ORG_RESULTS_AGGREGATOR_UPLOAD_USER }}
```

Comment:
I'm not sure what will be the behavior of Git Hub in forks here, if it will fail because the variables or secrets don't exist or just it will set an empty value. 

Can you create this same PR from your fork? I think that is the simplest way to test if the change will work with forks.

Reply:
Yes, lets try it out :)

Reply:
PR here - https://github.com/ansible/ansible-rulebook/pull/709

Reply:
After modifying my branch check to check if the variables are empty or not, this works.